diff -urN b/go/analysis/internal/facts/BUILD.bazel c/go/analysis/internal/facts/BUILD.bazel
--- b/go/analysis/internal/facts/BUILD.bazel	1969-12-31 16:00:00.000000000 -0800
+++ c/go/analysis/internal/facts/BUILD.bazel	2000-01-01 00:00:00.000000000 -0000
@@ -7,7 +7,7 @@
         "imports.go",
     ],
     importpath = "golang.org/x/tools/go/analysis/internal/facts",
-    visibility = ["//go/analysis:__subpackages__"],
+    visibility = ["//visibility:public"],
     deps = [
         "//go/analysis",
         "//go/types/objectpath",
@@ -17,7 +17,7 @@
 alias(
     name = "go_default_library",
     actual = ":facts",
-    visibility = ["//go/analysis:__subpackages__"],
+    visibility = ["//visibility:public"],
 )

 go_test(
